src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -103,12 +102,12 @@ public CitationRelationsTab(DialogService dialogService,
         this.databaseContext = databaseContext;
         this.preferences = preferences;
         this.libraryTab = libraryTab;
-        this.stateManager = stateManager;
         this.taskExecutor = taskExecutor;
         setText(Localization.lang("Citation relations"));
         setTooltip(new Tooltip(Localization.lang("Show articles related by citation")));
 
-        this.duplicateCheck = new DuplicateCheck(new BibEntryTypesManager());
+        this.entryTypesManager = new BibEntryTypesManager();
```

Comment:
You can pass the instance from the caller (see EntryEditor.java)

Reply:
Use the instance from EntryEditor as well. Mabe, this will also fix some bugs...